src/main/scala/gestalt/api.scala

```diff
@@ -67,6 +68,11 @@ object api extends Toolbox {
     Ident.apply("java")
   }
 
+  def Ident(name: "_no_package_")(implicit dummy: Dummy2): TermTree = {
+    import options.unsafe
+    Ident.apply("<empty>")
+  }
+
```

Comment:
I was considering `_empty_` initially, but `_no_package_` is more descriptive IMHO. The only argument for having `_empty_` is to match what Dotty internals have.
Will rebase and rebuild the PR shortly just to remove the excessive merges.